node/cn/api_tracer.go

```diff
@@ -792,6 +792,8 @@ func (api *PrivateDebugAPI) traceTx(ctx context.Context, message blockchain.Mess
 
 	case *tracers.Tracer:
 		return tracer.GetResult()
+	case *tracers.InternalTxTracer:
```

Comment:
How can this code be triggered? I can't see `NewInternalTxLogger` except for test file.

Reply:
To minimize the complexity of this PR, a feature to turn on the tracer will be added in another PR.